forespoersel-mottatt/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/forespoerselmottatt/ForespoerselMottattRiverTest.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.forespoerselmottatt
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.ints.shouldBeExactly
+import io.kotest.matchers.maps.shouldContainExactly
+import io.kotest.matchers.maps.shouldContainKey
+import io.mockk.clearAllMocks
+import kotlinx.serialization.json.JsonElement
+import no.nav.helse.rapids_rivers.testsupport.TestRapid
+import no.nav.helsearbeidsgiver.felles.BehovType
+import no.nav.helsearbeidsgiver.felles.EventName
+import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.json.toJson
+import no.nav.helsearbeidsgiver.felles.json.toMap
+import no.nav.helsearbeidsgiver.felles.rapidsrivers.pritopic.Pri
+import no.nav.helsearbeidsgiver.felles.test.rapidsrivers.firstMessage
+import no.nav.helsearbeidsgiver.felles.test.rapidsrivers.sendJson
+import no.nav.helsearbeidsgiver.utils.json.toJson
+import no.nav.helsearbeidsgiver.utils.test.wrapper.genererGyldig
+import no.nav.helsearbeidsgiver.utils.wrapper.Fnr
+import no.nav.helsearbeidsgiver.utils.wrapper.Orgnr
+import java.util.UUID
+
+class ForespoerselMottattRiverTest :
+    FunSpec({
+        val testRapid = TestRapid()
+
+        ForespoerselMottattRiver().connect(testRapid)
+
+        beforeEach {
+            testRapid.reset()
+            clearAllMocks()
+        }
+
+        test("Ved notis om mottatt forespørsel publiseres behov om notifikasjon") {
```

Comment:
Trenger vi ikke sjekke den andre av de to testene lenger? Altså `"Ved feil så republiseres den innkommende meldingen"`

Reply:
Nei, jeg fjernet den funksjonaltiteten. Den stammer fra gammelt av, og har tidvis vært ustabil. Jeg tror ikke den lenger er nødvendig.